bin/acquit-markdown.js

```diff
@@ -27,8 +28,31 @@ if (commander.require) {
   }
 }
 
-const contents = fs.readFileSync(commander.path).toString();
 const header = commander.header ? fs.readFileSync(commander.header, 'utf8') : '';
-const markdown = acquit.parse(fs.readFileSync(commander.path, 'utf8'));
+let markdown = '';
+
+if (fs.statSync(commander.path).isDirectory()) {
+  const getFiles = (dir, files_) => {
+    files_ = files_ || [];
+    const files = fs.readdirSync(dir);
+    for (const i in files){
+      const name = dir + '/' + files[i];
+      if (fs.statSync(name).isDirectory()){
+        getFiles(name, files_);
+      } else {
+        files_.push(name);
+      }
+    }
+    return files_;
+  };
+
+  getFiles(commander.path).forEach(file => {
+    markdown += acquit.parse(fs.readFileSync(file, 'utf8'));
+  });
+} else {
+  markdown = acquit.parse(fs.readFileSync(commander.path, 'utf8'));
+}
+
+markdown = markdown.replace(/eslint-\S*/g, '').replace(/TODO[\s\S]*$/g, '');
```

Comment:
I'm a little concerned about this line. Is it necessary? Stripping out 'TODO' seems like it's outside the scope of this tool

Reply:
I've used this tool to generate documentation for my tests and I was a little confused about TODO's in the middle of the generated text. So in my opinion there are could be few TODO's in your tests, but you don't want to see them in documentation

Reply:
Can we put it behind a flag? I'd rather not make this a default behavior - tricky to document. Maybe a `--replace` flag that takes in a regexp?